main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -2,130 +2,80 @@
 
 <Zoe-Version/>
```

Comment:
Can we update the Zoe version? It should be updated for 0.7.0. I can show you where it gets updated